common/src/main/java/com/wynntils/services/map/pois/TerritoryPoi.java

```diff
@@ -88,13 +89,16 @@ public void renderAt(
                 && McUtils.mc().screen instanceof GuildMapScreen guildMapScreen
                 && guildMapScreen.isResourceMode()) {
             color = territoryInfo.getResourceColor();
-        } else if (!isTerritoryInfoUsable() || territoryInfo.getGuildName().equals(territoryProfile.getGuild())) {
+        } else if (isTerritoryInfoUsable()) {
             // We know the guild name with it's color
-            color = territoryProfile.getGuildColor();
+            // Get the guild profile from the model, most likely it's already cached
```

Comment:
Seems these two if cases only differs in whether you do `territoryInfo.getGuildName()` or `territoryProfile.getGuild()`. Maybe extract this as a 
`guildName = isTerritoryInfoUsable() ? `territoryInfo.getGuildName() : territoryProfile.getGuild()`?

Reply:
I think I kept it this way to highlight the 3 cases: hybrid mode, advancement mode, and fallback. I don't remember fully though.

Reply:
I opened https://github.com/Wynntils/Artemis/pull/2110